app/src/main/res/values-ar/strings.xml

```diff
@@ -27,6 +27,10 @@
     <string name="keycode_shift">Shift</string>
     <string name="keycode_enter">Enter</string>
     <!-- Settings -->
+    <string name="keyboard_height_multiplier">Keyboard height</string>
+    <string name="settings_keyboard_height_multiplier_option_small">Small</string>
```

Comment:
havent checked the code yet, but you have these 3 strings in Simple Commons already